convert2rhel/toolopts.py

```diff
@@ -325,6 +362,19 @@ def _process_cli_options(self):
         if parsed_opts.org:
             tool_opts.org = parsed_opts.org
 
+        # Checks of multiple authentication sources
+        if tool_opts.password and tool_opts.activation_key:
+            loggerinst.warning("Set only one of password or activation key. Activation key take precedence.")
+
+        if parsed_opts.password and parsed_opts.password_from_file:
+            loggerinst.warning("Password file argument take precedence over the password argument.")
+
+        if (config_opts.activation_key or config_opts.password) and (parsed_opts.activationkey or parsed_opts.password):
+            loggerinst.warning("Command line authentication method take precedence over method in configuration file.")
+
+        if (config_opts.activation_key or config_opts.password) and parsed_opts.password_from_file:
+            loggerinst.warning("Password file take precedence over the config file.")
```

Comment:
I'd change this precedence for two reasons:
- The password file option is deprecated
- We prefer customers to use the activation key rather than the username/password combination as it's more user friendly. So if the user specifies both the AK and the password, I'd take the AK.

Reply:
Ok, what I've proposed above might be too big of a change considering the PR is already merged. So perhaps just changing the wording is enough:
```suggestion
loggerinst.warning("You have passed the RHSM credentials both through a config file and through a password file. We're going to use the password file.")
```